.github/workflows/ci.yml

```diff
@@ -288,6 +288,9 @@ jobs:
           - name: x86_64-gnu-aux
             os: ubuntu-20.04-4core-16gb
             env: {}
+          - name: x86_64-gnu-integration
+            os: ubuntu-20.04-16core-64gb
```

Comment:
Note: We should try to reduce builder sizes where it's not adding value. Historically 16 core builders rarely make things *that* much faster -- once this lands and we collect a few dozen builds we can re-evaluate, perhaps at the next bump point for the pinned commit.